src/main/java/net/logstash/logback/appender/AbstractLogstashTcpSocketAppender.java

```diff
@@ -506,25 +599,40 @@ public synchronized void start() {
             errorCount++;
             addError("No encoder was configured for appender " + name + ".");
```

Comment:
Some but not all error report the appender name in the message.
As far as I can tell, this is not needed as logback core already includes enough information about the failing appender when throwing its exception.
I propose to remove the appender name from every error messages generated by the appender.